# PARAMS for mission planner 

horizon_length: 40     # steps
n_drones: 2            # drones
leader_id: 1           # drone_id
step_size: 0.2         # seconds
planning_rate: 2       # sec
visualization_rate: 3  # sec
clock_rate: 0.01       # sec
topics_rate: 1         # sec
vel_max: 0.7           # m/s
acc_max: 1             # m/sÂ²
inspection_dist: 6     # m 
inc_distance: 0.2      # m
inc_angle: 0.025       # rad

frame: "map"
